src/lua/color_display_480x272/TOOLS/elrsV2.lua

```diff
@@ -341,7 +341,7 @@ end
 
 local function fieldFolderDisplay(field,y ,attr)
   lcd.drawFilledRectangle(0, y, LCD_W, 22, TITLE_BGCOLOR)
```

Comment:
I think we should just remove to keep consistency with the LUA for other devices that has a problem that they are too over-highlighting.